ibis/backends/duckdb/__init__.py

```diff
@@ -48,32 +48,45 @@ def _quote(name: str):
 
 
 @_generate_view_code.register(r"parquet://(?P<path>.+)", priority=10)
-def _parquet(_, path, table_name=None):
+def _parquet(_, path, table_name=None, **kwargs):
     path = Path(path).absolute()
     table_name = table_name or _name_from_path(path)
     quoted_table_name = _quote(table_name)
+    kw = ""
+    if kwargs:
+        kw = "," + ",".join(f"{k}={v}" for k, v in kwargs.items())
```

Comment:
I wonder if

```suggestion
    args = [repr(path)]
    args.extend(f"{k}={v}" for k, v in kwargs.items())
    
    # github doesn't let me suggest things for deleted lines
    # so you'd have to sub in `from read_parquet(", ".join(args))` too
```

is clearer.